f2/src/forms/MoneyLostInfoForm.js

```diff
@@ -146,6 +147,7 @@ export const MoneyLostInfoForm = (props) => {
                                 component={Input}
                                 onBlur={handleBlur}
                                 onChange={handleChange}
+                                id={'checkbox-' + question.name}
```

Comment:
This is not correct, it will result in two elements with the same ID - "checkbox-other".

Reply:
The formOptions object does not contain descriptionLabel or descriptionHelpText. If there is no need for a label (I think we should consider some sort of label) we could use the `HiddenText` component.

Reply:
will remove this id line